reviews/graduation-containerd.md

```diff
@@ -0,0 +1,62 @@
+
+In late 2017, containerd announced its [1.0 release](https://blog.docker.com/2017/12/cncf-containerd-1-0-ga-announcement/), followed in early 2018 by a 1.1 release that has been updated with several minor fix releases through 2018. The containerd team has just completed the next major release, [1.2.0](https://github.com/containerd/containerd/releases/tag/v1.2.0), with more complete Windows runtime support and a new runtime shim v2 API that is [proving](https://www.cncf.io/blog/2018/09/17/gsoc-18-kata-containers-support-for-containerd/) [valuable](https://twitter.com/resouer/status/1035066629887905792) for [runc](https://github.com/opencontainers/runc) alternatives like [Kata containers](https://katacontainers.io/).
+
+At this time, we believe containerd is ready for the [graduation stage](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage) within the CNCF, and per the guidelines listed there, we detail our readiness in the section below.
+
+### CNCF Graduation Criteria
+
+**Committers from at least two organizations.**
+
+Containerd has had a variety of maintainers and reviewers since its inception, and currently have 12 committers representing Docker, NTT, Google, IBM, Microsoft, Facebook, Tesla, and Cruise Automation. We also recognize **LGTM** rights for a group we call *reviewers*, of which there are currently six reviewers representing ZTE, Huawei, Docker, Microsoft and an independent developer.
+
+**Have achieved and maintained a Core Infrastructure Initiative Best Practices Badge.**
+
+We have recently validated our CII best practices, and have the badge embedded within our main containerd/containerd repository on GitHub. Our specific status (passing) is shown on [our CII project page here](https://bestpractices.coreinfrastructure.org/en/projects/1271).
+
+**Adopt the CNCF Code of Conduct.**
+
+We have adopted the CNCF code of conduct and prominently make this clear in our [GOVERNANCE.md](https://github.com/containerd/project/blob/master/GOVERNANCE.md) document, which applies to all containerd projects and sub-projects.
+
+**Explicitly define a project governance and committer process.**
+
+Our project governance is clearly laid out in our [GOVERNANCE.md](https://github.com/containerd/project/blob/master/GOVERNANCE.md) document, including details on how to become a maintainer and how maintainer and contribution processes are handled. The maintainers for containerd, the CRI project, and all sub-projects are common, and maintained in our core project repo in the [MAINTAINERS](https://github.com/containerd/project/blob/master/MAINTAINERS) file.
+
+**Have a public list of project adopters for at least the primary repository.**
```

Comment:
Can we also add https://github.com/firecracker-microvm/firecracker-containerd to the adopter list

Reply:
Added, thanks!